packages/assets-controllers/src/TokenDetectionController.ts

```diff
@@ -226,6 +327,15 @@ export class TokenDetectionController extends StaticIntervalPollingController<
     this.#disabled = true;
   }
 
+  /**
+   * Internal isActive state
+   *
+   * @type {object}
+   */
+  get isActive() {
```

Comment:
Are there any tests we need to add for this?

Reply:
I'm okay with the tests being added in #3867, so I'm resolving this conversation.